pages/data-design/branches/pulling-pushing-items-between-branches.mdx

```diff
@@ -0,0 +1,113 @@
+- [**Pulling an item**](#pulling-items-from-another-branch-coming-in-march-2025): Adding an item to a branch that the user is currently on, from another branch.
+
+<Callout type="info">
+    Pushing is currently out in alpha and pulling will be available in March 2025. [Contact us](mailto:beta@avo.app)
+ to get access.
+</Callout>
+
+## Pushing items to another branch (alpha)
+
+Pushing events, properties and property bundles from the branch that a user is currently on, to another branch is possible from the Tracking plan changes view. 
+To access the tracking plan changes view, first you have to be on a branch and then you can click "View Changes" under the branch name.
+
+![Navigating to tracking plan changes](/images/pulling-pushing-items/navigating-to-tracking-plan-changes.png)
+
+There is an button in the top right corner of every tracking plan change that can be pushed to another branch.
+
+![Pressing push button](/images/pulling-pushing-items/press-push-button.png)
+
+When that button is pushed, you will get a modal that prompts you to pick a branch to push the changes to and review your selection.
+You have to first pick a branch so that Avo can accurately display how the changes will be applied, as it might differ per branch if the item already exists on the destination branch.
+
+<center><img
+  src="/docs/images/pulling-pushing-items/empty-push-modal.png"
+  width="60%"
+  alt="Empty review modal"
+/></center>
+
+When the destination branch is selected, the actual diff for the destination branch is shown. The items shown can be expanded to show the details of what will be added to the destination branch.
+If the item you selected to push is an event or a property bundle, any related property changes are displayed below. See the [property changes related to event](#property-changes-related-to-event) section for details.
+Once you have reviewed your selection, you can press the Push to branch button and the changes will be applied to the destination branch.
+
+<center><img
+  src="/docs/images/pulling-pushing-items/collapsed-diff-modal.png"
+  width="60%"
+  alt="Empty review modal"
+/></center>
+
+Then you can navigate to your destination branch and use the item you pushed there.
+
+![Empty review modal](/images/pulling-pushing-items/destination-branch.png)
+
+## Pulling items from another branch (coming in March 2025)
+
+Soon it will be possible to search and get results from other branches. Those item can be pulled to the branch the user is on, and can be used on that branch.
+
+## Merging branches where items have been pushed or pulled
+
+When either the source or destination branch where items have been pushed or pulled are merged to main, you will be prompted to pull main on the branch where the items still remain.
+As the ID of the items are the same, the items exist on main and there is no need to keep them specifically on the other branch. 
```

Comment:
And something like "...however changes to these items between the branches will reconcile and the remaining diff will appear instead of the item"